### PR TITLE
flake8 test passing

### PR DESCRIPTION
**Pull request by tusharmakkar08**
Hey @brian-brazil 

Earlier flake8 test were failing. 
screen shot 2019-02-15 at 8 01 45 pm

I have fixed the tests.
screen shot 2019-02-15 at 7 56 12 pm

P.S: This is my first PR on this repo. Let me know if I have missed anything. 
Thanks.

### CODE REVIEW

File: tests/test_parser.py [hunk 1]
```diff
@@ -241,18 +233,9 @@ def test_label_escaping(self):
...
-                ('\\\\foo', '\\foo'),
-                ('foo\\', 'foo\\'),
-                ('foo\\\\', 'foo\\'),
-                ('\\n', '\n'),
-                ('\\\\n', '\\n'),
-                ('\\\\\\n', '\\\n'),
-                ('\\"', '\\"'),
-                ('\\\\"', '\\"'),
-                ('\\\\\\"', '\\\\"')]:
+        for escaped_val, unescaped_val in [('foo', 'foo'), ('\\foo', '\\foo'), ('\\\\foo', '\\foo'),
```
Comment (brian-brazil) [inline tests/test_parser.py]: This was clearer as it was

File: prometheus_client/core.py [hunk 2]
```diff
@@ -2,11 +2,8 @@
 
 from .metrics import Counter, Enum, Gauge, Histogram, Info, Summary
 from .metrics_core import (
-    CounterMetricFamily, GaugeHistogramMetricFamily, GaugeMetricFamily,
-    HistogramMetricFamily, InfoMetricFamily, Metric, Sample,
-    StateSetMetricFamily, SummaryMetricFamily, UnknownMetricFamily,
-    UntypedMetricFamily,
-)
+    CounterMetricFamily, GaugeHistogramMetricFamily, GaugeMetricFamily, HistogramMetricFamily, InfoMetricFamily,
```
Comment (brian-brazil) [inline prometheus_client/core.py]: This was changed from this before on style grounds, is there a standard here?

Comment (tusharmakkar08) [reply prometheus_client/core.py]: I am not sure about the standard here but after manually changing the flake8 comments on each file, I applied the auto-formatter from pycharm which yielded this result.

File: tests/test_parser.py (hunk 1 above)
Comment (tusharmakkar08) [reply tests/test_parser.py]: Agree but flake8 gives following error:

`tests/test_parser.py:232:5: E129 visually indented line with same indent as next logical line`

Should I add E129 in flake8 ignore? Let me know how to proceed @brian-brazil

Comment (tusharmakkar08) [reply tests/test_parser.py]: Updated it.

**Comment by tusharmakkar08** [comment]
Hey @brian-brazil 

Did the corresponding changes to make the PR compliant. 

Thanks.

File: prometheus_client/core.py (hunk 2 above)
Comment (brian-brazil) [reply prometheus_client/core.py]: I don't think it's consistent with what's in the tox.ini

Comment (tusharmakkar08) [reply prometheus_client/core.py]: Tox.ini has flake8 where we aren't ignoring ordering of imports. If we want to ignore that, please let me know.

Comment (brian-brazil) [reply prometheus_client/core.py]: Let's just make sure everything is consistent

Comment (tusharmakkar08) [reply prometheus_client/core.py]: Everything is consistent rn according to flake8.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**